node-hub/dynamixel-client/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "dynamixel-client"
+version = "0.1"
+authors = ["Hennzau <dev@enzo-le-van.fr>"]
+description = "Dora Node client for dynamixel motors."
+readme = "README.md"
+
+packages = [
+    { include = "main.py", to = "dynamixel_client" }
+]
+
+[tool.poetry.dependencies]
+dora-rs = "0.3.5"
+
+common = { path = "../../common" }
```

Comment:
So this dependency will make it hard to pip install this package independently from this repo?